### DISCUSSION TITLE
How do I play an MP4 on Github without downloading the file?

### DISCUSSION THREAD

**Discussion by zqstack** [community/community#39956] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Show & Tell

### Body

On Github I have an MP4 file, which I can download or view as RAW. Is there a way to play / view it from the browser without downloading it?

I've tried embedding it into a wiki page or README.md file, without success. It's meant to be a private file, to show other developers instructions for the repository, so I would like to avoid using YouTube to play it

**Comment by happy-spark** [comment, marked as answer]
Probably not. I think that's a necessary function

**Comment by DrakeYounger** [comment]
You can use the "raw.githack.com" service. Just copy the link to your MP4 file in the repository, and paste it into the "Enter a Github URL to have it served through raw.githack.com" field on their website. It'll generate a new link, and when you open it in your browser, voila! Your video plays smoothly without any downloads. Also, you can use [video editing software](https://www.movavi.com/) to change it or convert it into another file. I hope that helps!

**Reply by Gustavohps10** [reply]
I combined "raw.githack.com" with "livepush.io/mp4-player/" and that's it, it's working perfectly without needing to download
